tests/build/build.go

```diff
@@ -515,6 +515,51 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to finish")
 			})
 
+			It("should validate Tekton Results", func() {
+				// Create an Service account and a token associating it with the service account
+				resultSA := "tekton-results-tests"
+				_, err := f.AsKubeAdmin.CommonController.CreateServiceAccount(resultSA, testNamespace, nil)
+				Expect(err).NotTo(HaveOccurred())
+				_, err = f.AsKubeAdmin.CommonController.CreateRoleBinding("tekton-results-tests", testNamespace, "ServiceAccount", resultSA, "ClusterRole", "tekton-results-readonly", "rbac.authorization.k8s.io")
+				Expect(err).NotTo(HaveOccurred())
+
+				resultSecret := &v1.Secret{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:        "tekton-results-tests",
+						Annotations: map[string]string{"kubernetes.io/service-account.name": resultSA},
+					},
+					Type: v1.SecretTypeServiceAccountToken,
+				}
+
+				_, err = f.AsKubeAdmin.CommonController.CreateSecret(testNamespace, resultSecret)
+				Expect(err).ToNot(HaveOccurred())
+				err = f.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(testNamespace, resultSecret.Name, resultSA)
+				Expect(err).ToNot(HaveOccurred())
+
+				resultSecret, err = f.AsKubeAdmin.CommonController.GetSecret(testNamespace, resultSecret.Name)
+				Expect(err).ToNot(HaveOccurred())
+				token := resultSecret.Data["token"]
+
+				fmt.Println("token=", string(token))
```

Comment:
I guess we probably don't want token to be printed out?

Reply:
fixed